discoveries/air-quality-so2-volcanoes.discoveries.mdx

```diff
@@ -0,0 +1,162 @@
+---
+featuredOn:
+  - air-quality
+id: 'so2-volcanoes'
+name: Monitoring Volcanic Sulfur Dioxide Emissions
+description: "How NASA monitors volcanic emissions impact on global air quality"
+media:
+  src: ::file ./volcano--discovery-cover.jpg
+  alt: Ash cloud over mountain
+  author:
+    name: Yosh Ginsu
+    url: https://unsplash.com/photos/qexZLgMcbPc
+pubDate: 2022-11-04
+thematics:
+  - air-quality
+---
+<Block>
+  <Prose>
+    Watching lava and magma flow from an erupting volcano may seem fascinating at first sight. But volcanoes emit dangerous gasses, including [sulfur dioxide](https://so2.gsfc.nasa.gov/so2intro.html) (SO₂), which can be toxic to the human body and the environment. 
+
+    The most significant natural contributor to sulfur emissions is volcanic degassing, or when large amounts of gas escape through the soil near volcanoes. Volcanic gases are classified as air pollutants in many countries because of their effects on the human respiratory system. 
+    
+    </Prose>
+    <Figure>
+    <Image
+      src={new URL("./USGS_Leilani_Estates_Hawaii_Volcanic_EruptionDSC_0411.jpg", import.meta.url).href}
+      alt="U.S. Geological Survey (USGS) volunteer Allan Lerner uses hydrogen sulfide (H2S) and sulfur dioxide (SO2) sensors to test the air quality after the Kīlauea volcanic eruption. "
+    />
+    <Caption
+      attrAuthor="Grace Simoneau/FEMA"
+      attrUrl="https://commons.wikimedia.org/wiki/File:20180519_USGS_Leilani_Estates_Hawaii_Volcanic_EruptionDSC_0411_medium.jpg"
+      >
+      Testing air quality on Kīlauea
+      </Caption>
+  </Figure>
+</Block>
+
+<ScrollytellingBlock>
+  <Chapter
+    center={[0, 0]}
+    zoom={2}
+    datasetId='so2'
+    layerId='OMSO2PCA-COG'
+    datetime='2005-01-01'
+  >
+    ## Monitoring air pollution with satellites
+    
+    Collecting measurements of sulfur dioxide plumes from space is a valuable way to monitor changes in emissions.
+  </Chapter>
+  <Chapter
+    center={[-92.049, -0.7531]}
+    zoom={4}
+    datasetId='so2'
+    layerId='OMSO2PCA-COG'
+    datetime='2005-01-01'
+  >
+    Tracking new and existing sources of SO2 is crucial because it affects environmental processes that directly impact our health, the global ecosystem, atmospheric chemistry, and climate. SO2 irritates the eyes, nose, and lungs. High concentrations of SO2 can result in temporary breathing impairment. Particle deposition from SO2 can damage stone and other materials, including culturally and historically significant objects such as statues and monuments. It can lead to acid rain, damaging foliage and hindering future growth. Volcanic-sourced SO2 contributes between one-quarter and one-third of total SO2 emissions. _Shown on the map: 2005 volcanic activity in the Galápagos islands._
+  </Chapter>
+  <Chapter
+    center={[157.938876, 55.583288]}
+    zoom={4}
+    datasetId='so2'
+    layerId='OMSO2PCA-COG'
+    datetime='2009-09-01'
```

Comment:
@kcarini @aboydnw The 2009 dataset is not loading because the date is incorrect. Being an yearly dataset, the data falls on the 1st of january, so the date should be `2009-01-01`, like you've one with all the other chapters.

```suggestion
    datetime='2009-01-01'
```